pkgs/top-level/all-packages.nix

```
@@ -19845,6 +19845,8 @@ with pkgs;
   modd = callPackage ../development/tools/modd { };
 
   mold = callPackage ../development/tools/mold { };
+  
+  mold-unwrapped = callPackage ../development/tools/mold-unwrapped { };
```

Can be simplified to just

```suggestion
  mold = wrapBintoolsWith { bintools = mold-unwrapped; };
  
  mold-unwrapped = callPackage ../development/tools/mold-unwrapped { };
```